lib/organisms/components/faqs/faq2/index.tsx

```diff
@@ -27,17 +33,14 @@ const FaqSection = ({
   buttonWrapper,
   button,
   model,
-}) => {
-  let data = model.data.reduce(function(map, obj) {
-    map[obj.Key] = obj.Value;
-    return map;
-  }, {});
+  data,
+}:FaqSectionProps) => {
   return (
     <FaqSectionWrapper id={model.name}>
       <Container>
         <Box {...sectionHeader}>
-          <Text {...sectionSubTitle} content={data.subTitle} />
-          <Heading {...sectionTitle} content={data.title} />
+          <Text {...sectionSubTitle} content={data.title} />
+          <Heading {...sectionTitle} content={data.subTitle} />
```

Comment:
```suggestion
          <Text {...sectionSubTitle} content={data.subTitle} />
          <Heading {...sectionTitle} content={data.title} />
```

Shouldn't we keep it like it was? If there is a problem with the data, maybe you need to change it in DConfig.
Because I feel Heading appears as the leading title while the text would be less emphasized

Reply:
That is, there is a mistake in showing them..
I told you that I can't edit anything with Dconfig. I can't access my account and it gives me an error. I filmed a video for you and sent it to you. I'm waiting for you to fix the problem. In order for me to edit the data and the routing in it, I have to modify it.

I will return it as it was and only the Dconfig problem will be solved ..I will adjust the data

Reply:
I will return it as it was